package.json

```diff
@@ -63,11 +61,12 @@
     "jest": "^26.0.1",
     "prettier": "^2.0.5",
     "prop-types": "^15.7.2",
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
     "rimraf": "^3.0.0",
     "rollup": "^2.9.1",
-    "rollup-plugin-peer-deps-external": "^2.2.0"
+    "rollup-plugin-peer-deps-external": "^2.2.0",
+    "@wojtekmaj/enzyme-adapter-react-17": "^0.6.6"
```

Comment:
this looks like a private fork and package, isn't the official package ready yet? If not, I wouldn't rush with the upgrade

Reply:
> this looks like a private fork and package, isn't the official package ready yet? If not, I wouldn't rush with the upgrade

I did a bit of research, an at first glace it looks like they just gave up. The issue for supporting React 17 https://github.com/enzymejs/enzyme/issues/2429 since almost 2 years now, while the fork we chose here looks like it's seriously maintained and widely adopeted (550k weekly download, the activity on the repo looks legitimate: https://github.com/wojtekmaj/enzyme-adapter-react-17). At this point the choice is either trust this fork or give up on having enzyme.

Reply:
Yes I agree, enzyme feels kind of dead, so unless you want to move to something else altogether for testing I think using the fork is the way to go